src/StatusesHandler.php

```diff
@@ -0,0 +1,270 @@
+<?php
+
+namespace RebelCode\Bookings\WordPress\Module;
+
+use Dhii\Invocation\InvocableInterface;
+use Dhii\Util\Normalization\NormalizeArrayCapableTrait;
+use Psr\EventManager\EventManagerInterface;
+use RebelCode\Modular\Events\EventsConsumerTrait;
+use Psr\EventManager\EventInterface;
+use Dhii\Event\EventFactoryInterface;
+use Traversable;
+
+/**
+ * Handler for providing the status related state and functionality.
+ *
+ * @since [*next-version*]
+ */
+class StatusesHandler implements InvocableInterface
```

Comment:
What is the result of applying this handler to a request, in general?

Reply:
This question is about class name? I mean in this version this class name doesn't descriptive enough and in the new version I should fix it?

Reply:
See previous answer to similar comment.